.storybook/DocsDecorator/HealthFactors/SingleDoc/index.styles.ts

```diff
@@ -14,9 +14,10 @@ const styles = {
 
   linkIconContainer: (theme: Theme) =>
     css({
-      width: `${theme.spacings.DOUBLE}rem`,
-      height: `${theme.spacings.DOUBLE}rem`,
-      marginLeft: `${theme.spacings.HALF}rem`,
+      width: `${pixelsToRem(14)}rem`,
+      height: `${pixelsToRem(14)}rem`,
+      marginLeft: '0.5em',
```

Comment:
Should this be em rather than rem?

Reply:
Yes, changed to em

Reply:
To be clear, this one has a value of 'em' just wondering why when the rest are 'rem'.

Reply:
Webcore uses em to space their icons from the health factor text. I am assuming that their UX team wanted to scale the spacing in proportion of the health factors' font size. Not sure the reason of this choice, but I followed their design.